docs/how-to-guides/environment-setup-and-maintenance/adding-usgscsm-plugin-to-isis.md

```diff
@@ -0,0 +1,78 @@
+# Adding USGSCSM Plugin to ISIS
+
+## For users
+You can install the latest [USGSCSM](https://github.com/DOI-USGS/usgscsm) library via conda:
+
+```sh
+conda install conda-forge::usgscsm
+```
+## For developers
+
+### Plugin setup
+Update `IsisPreferences` file under `Group = Plugins` and add `"$CONDA_PREFIX/lib/csmplugins/"`, for example:
+
+  ```
+  Group = Plugins
+    CSMDirectory = ("$CONDA_PREFIX/lib/csmplugins/", -
+                    "$ISISROOT/lib/csmplugins/", -
+                    "$ISISROOT/lib/isis/csm3.0.3/", -
+                    "$ISISROOT/csmlibs/3.0.3/", -
+                    "$HOME/.Isis/csm3.0.3/")
+  EndGroup
+  ```
+
+??? info
+    If you want to find where the installation path is being defined, look [here](https://github.com/DOI-USGS/usgscsm/blob/main/CMakeLists.txt#L712).
```

Comment:
I feel linking the specific line is asking for this to be out of date really fast. I think metioning "for USGSCSM versions X and above the plugin is installed in `$CONDA_PREFIX/lib/csmplugins/` in an Anaconda environment." Or something along those lines as we don't expect it to change again anytime soon.